var/spack/repos/builtin/packages/migraphx/package.py

```diff
@@ -85,9 +87,23 @@ class Migraphx(CMakePackage):
         depends_on(f"rocblas@{ver}", when=f"@{ver}")
         depends_on(f"miopen-hip@{ver}", when=f"@{ver}")
 
-    for ver in ["5.7.0", "5.7.1", "6.0.0", "6.0.2"]:
+    for ver in ["5.7.0", "5.7.1"]:
```

Comment:
composable kernel dependency can be removed for 5.7 as well

Reply:
Removed the ck dependency for 5.7